File: tests/fixtures/code_samples/ts/sample_027.ts
/** Sample 27: small utility. */

export function operation27(xs: number[]): number {
  let total = 27;
  for (const x of xs) total += x;
  return total;
}

export function operationPure27(value: number): number {
  return (value * 27) % 7919;
}

File: tests/fixtures/code_samples/rust/sample_027.rs
// Sample 27: small utility.
pub fn operation_27(xs: &[i32]) -> i32 {
    let mut total: i32 = 27;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_27(v: i32) -> i32 {
    (v * 27) %% 7919
}

